### ISSUE TITLE
Request: history to be separate for each container

### ISSUE THREAD

**Issue by jbyler** [mozilla/multi-account-containers#1233]
I would like to request that history be made per-container, so that the history for one container is separate from the history for another container.  The omnibar should only show URLs from the history of the current container.  This is quite important to my use case.  I use per-account containers to keep separate browsing contexts, and I don't want URLs from different contexts showing up in the omnibar.

For some reason I had assumed this was the case with per-account containers already.  I just realized it's not, and I think it reduces the usefulness of this feature significantly.  Please consider adding this!

**Comment by Vinnl** [comment]
See #537; click the Thumbs Up below the request to vote for it.

**Comment by kendallcorner** [comment]
Yes. This looks like a duplicate of #537. Please vote (:+1:) or discuss on that issue to show your interest. I'll close this, but feel free to reopen if you think it's a different issue.

**Comment by Vinnl** [comment]
@kendallcorner I think this (and #1544) are somewhat different from #537: this issue wants the history to be completely isolated from other containers, while my request in #537 was merely to use it as a signal in the ranking, i.e. sites visited in another container would still be suggested, but just less prominently than sites often visited in the current container.

**Comment by Fabian42** [comment]
Related (and would likely be fixed together): #1826 
I think a solution that brings together this proposal and #47 would be best.

**Comment by anewuser** [comment]
This is a duplicate of #952.

**Comment by ajmv78** [comment]
Agreed. If your browsing history is still shared, using a different browser for specific personas is still a more tidy solution. 

+1 on this request.
